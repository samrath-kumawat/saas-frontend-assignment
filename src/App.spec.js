import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
test('match snapshots', async () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
});