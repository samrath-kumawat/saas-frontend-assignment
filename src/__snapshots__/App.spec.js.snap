// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App match snapshots 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="container"
    >
      <table
        class="tableBody"
      >
        <thead>
          <tr
            class="tableHeaderRow"
          >
            <th
              class="tableHeader thFirstCellRadius"
            >
              S.No.
            </th>
            <th
              class="tableHeader"
            >
              Percentage Funded
            </th>
            <th
              class="tableHeader thLastCellRadius"
            >
              Amount Pledged
            </th>
          </tr>
          <tr />
          <tr />
        </thead>
        <tbody />
      </table>
    </div>
  </div>
</DocumentFragment>
`;
